inference-engine/thirdparty/clDNN/src/gpu/custom_task_arena.cpp

```diff
@@ -0,0 +1,268 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom_task_arena.h"
```

Comment:
can we put custom arena to static library and reuse in both IE / clDNN?
Code duplication here is not accepted

Reply:
Or can we use high level CPUStreamExecutor here?

Reply:
We cannot use CPUStreamExecutor simply, because we need to use task_arena  at LoadNetwork stage while CPUStreamExecutor creates threads at Infer stage.

Reply:
After offline discussion=> We'll use CPUStreamExecutor in our next PR targeting master branch

Reply:
(And I think "put custom arena to static library and reuse it both IE/clDNN" is good, too. Lets discuss again about that)